platforms/karaf/features/src/main/resources/features.xml

```diff
@@ -1465,6 +1465,13 @@
     <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.jing/${jing-bundle-version}</bundle>
     <bundle>mvn:org.apache.camel/camel-jing/${project.version}</bundle>
   </feature>
+  <feature name='camel-jira' version='${project.version}' start-level='50'>
+    <feature version='${project.version}'>camel-core</feature>
+    <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.ant/${ant-bundle-version}</bundle>
+    <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.isorelax/${isorelax-bundle-version}</bundle>
+    <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.jira-rest/${jira-rest-bundle-version}</bundle>
```

Comment:
This jira-rest bundle doesn't exist.

http://repo2.maven.org/maven2/org/apache/servicemix/bundles/

This looks like the old camel-jira feature? We need a new one, I guess. I think the guava version used in the component will create problem to the feature.

Reply:
There is no jira bundle there, so I removed the entry from xml.